lms/static/js/spec/discovery/discovery_spec.js

```diff
@@ -0,0 +1,587 @@
+define([
+    'jquery',
+    'sinon',
+    'backbone',
+    'logger',
+    'js/common_helpers/template_helpers',
+    'js/discovery/app',
+    'js/discovery/collection',
+    'js/discovery/form',
+    'js/discovery/result',
+    'js/discovery/result_item_view',
+    'js/discovery/result_list_view',
+    'js/discovery/filter',
+    'js/discovery/filters',
+    'js/discovery/filter_bar_view',
+    'js/discovery/filter_view',
+    'js/discovery/search_facets_view',
+    'js/discovery/facet_view',
+    'js/discovery/facets_view'
+], function(
+    $,
+    Sinon,
```

Comment:
You might find these helper functions useful: https://github.com/edx/edx-platform/blob/master/common/static/js/spec_helpers/ajax_helpers.js